src/classes/BDI_DataImport_API.cls

```diff
@@ -119,7 +119,7 @@ global with sharing class BDI_DataImport_API {
     * @return void
     */
     global static void processDataImportRecords(Data_Import_Settings__c diSettings, List<DataImport__c> listDI, Boolean isDryRun) {
-        BDI_DataImportService bdi = new BDI_DataImportService(isDryRun);
+        BDI_DataImportService bdi = new BDI_DataImportService(isDryRun, BDI_FieldMappingHelpText.getInstance());
```

Comment:
This global method explicitly uses help text mapping and will not use advanced mapping even when enabled in the org.  Should we consider documenting this change?

Reply:
So this is actually not a change in behavior, as previous to this release help text mapping was the only option- the thought here was to force API calls to use help text mapping at this point, so as not to change that behavior in an unpredictable way (our docs don't cover possible impacts to API calls I think).

Reply:
👍🏽